core/ir/x86/encode.c

```diff
@@ -236,6 +237,70 @@ const reg_id_t dr_reg_fixer[] = {
     DR_REG_BND2,    DR_REG_BND3,
 };
 
+/* Maps containing-registers to their DR_ISA_REGDEPS virtual-register.
+ * Same size as dr_reg_fixer[], keep them synched.
+ * Note: DR_REG_INVALID (== DR_REG_V187 == 187) has been excluded to avoid failures on
```

Comment:
We can, but what's the advantage?
If you're thinking about the jumps you mentioned here: https://github.com/DynamoRIO/dynamorio/pull/6783#discussion_r1571407194 this would not solve the "problem", the jumps happen because of the mapping to containing registers (same as they happen in dr_reg_fixer[]).

Reply:
Mhm, I guess the advantage is that all DR_REG_V would be valid (we'd still have a "jumps" in dr_reg_virtual[] though, that really depends on the mapping to containing registers).

Reply:
Done.